contributing/development/compiling/compiling_for_ios.rst

```diff
@@ -86,6 +88,13 @@ should be placed in ``libgodot.ios.debug.xcframework`` and ``libgodot.ios.releas
 The MoltenVK static ``.xcframework`` folder must also be placed in the ``ios_xcode``
 folder once it has been created.
 
+You can then zip the contents of the ``ios_xcode`` folder to reproduce the ``ios.zip``
```

Comment:
4.3 adds `generate_bundle=yes` SCons option to do it automatically, you do not need to copy, lipo, or zip anything anymore, just build all required architectures and add it to the last one you are building, e.g.,:

```
scons platform=ios target=template_release arch=arm64
scons platform=ios target=template_release arch=x86_64 ios_simulator=yes
scons platform=ios target=template_release arch=arm64 ios_simulator=yes
scons platform=ios target=template_debug arch=arm64
scons platform=ios target=template_debug arch=x86_64 ios_simulator=yes
scons platform=ios target=template_debug arch=arm64 ios_simulator=yes generate_bundle=yes
```

The same is also true for macOS templates (generates `.zip`) and macOS editor (generates signed `.app`).

Reply:
Good to know! Thanks for the info, I'll close this out then.